src/includes/performance/connect-errors-spans/java.mdx

```diff
@@ -32,6 +33,7 @@ try {
 import io.sentry.Sentry
 
 val span = Sentry.startTransaction("task")
+Sentry.configureScope { it.setTransaction(span) }
```

Comment:
let's name the lambda too, it makes a bit more idiomatic

Reply:
You mean `Sentry.configureScope { scope -> scope.setTransaction(span) }`?

Reply:
yes, example:
https://docs.sentry.io/platforms/java/#configure

we name it `options`